### PR TITLE
feat(renovate): use service account to merge renovate automerge prs (VF-3507)

### PR DESCRIPTION
**Pull request by Xavientois**
**Fixes or implements VF-3507**

### Brief description. What is this change?

Test for service account strategy to merge renovate automerge PRs with bors.

### Checklist

- [ ] this is a breaking change and should publish a new major version
- [ ] appropriate tests have been written

### CODE REVIEW

**Comment by Xavientois** [comment]
service-account merge

**Comment by Xavientois** [comment]
service-account merge

**Comment by Xavientois** [comment]
service-account merge

**Comment by Xavientois** [comment]
service-account merge

**Comment by Xavientois** [comment]
tested here: https://github.com/voiceflow/poc-autorebase-gh-action/pull/95#issuecomment-1165768235

File: .github/workflows/renovate-delegate.yaml [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
     runs-on: ubuntu-latest
     steps:
-      - name: Bors Delegate
+      - name: Merge Comment
         uses: thollander/actions-comment-pull-request@v1
         with:
-          message: bors delegate+
-          GITHUB_TOKEN: ${{ secrets.GH_SA_TOKEN }}
\ No newline at end of file
+          message: bors merge
```
Comment (xavidop) [inline .github/workflows/renovate-delegate.yaml:17]: can we change this for bors r+?

**Review by xavidop — APPROVED** [review]

**Pull request merged**